Project-Hotpot/Project-Hotpot/ConnectViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  ConnectViewController.swift
+//  Project-Hotpot
+//
+//  Created by Harini Sundaram on 7/8/22.
+//
+
+import UIKit
+//import "APIManager.swift"
+
+class ConnectViewController: UIViewController {
+
+   // MARK: - Subviews
+   let stackView = UIStackView()
+   let connectLabel = UILabel()
+   let connectButton = UIButton(type: .system)
+
+   var api_instance = APIManager.shared()
+    
+   // MARK: App Life Cycle
+   override func viewDidLoad() {
+       //initialize APIManager Object
+       
+       super.viewDidLoad()
+       style()
+       layout()
+   }
+    
+    @objc func didTapConnect(_ button: UIButton) {
+        guard let sessionManager = api_instance.sessionManager else { return }
+        sessionManager.initiateSession(with: scopes, options: .clientOnly)
+        
+        let storyBoard : UIStoryboard = UIStoryboard(name: "Main", bundle:nil)
+         let nextViewController = storyBoard.instantiateViewController(withIdentifier: "HomeNavController") as! UINavigationController
```

Comment:
I realize you're still prototyping, but wanted to comment early on that any use of the `!` coalescing operator for Optionals is strongly discouraged.
Prefer `guard let` or `if let` instead; this will allow you to gracefully handle potential nil values without silencing the compiler.
I make a big point of this because proper use of Optionals and non-Optionals saves me from myself on a daily basis 🙂